ls1intum/Apollon2
# 26 show class component

### Checklist


### Motivation and Context




This PR completes https://github.com/ls1intum/Apollon2/issues/26 
As a user, I want to see class diagram elements

### Description



With this pr, class diagram elements are rendered on canvas with their svg structures
Added Nodes:
- Package
- Class
- Color description

### Steps for Testing



1. Clone the project
2. Build the app from the root `npm run build`
3. Start the app `npm run start`
4. Observe Package and Class nodes in the Canva

### Screenshots





<img width="500" alt="Screenshot 2024-12-27 at 14 58 37" src="https://github.com/user-attachments/assets/9b6c0269-17c7-4177-a953-8ce449bc0881" />
